src/libraries/Microsoft.PowerFx.Core/Glue/SimpleResolver.cs

```diff
@@ -99,7 +99,7 @@ public virtual bool Lookup(DName name, out NameLookupInfo nameInfo, NameLookupPr
             return false;
         }
 
-        public IEnumerable<TexlFunction> LookupFunctions(DPath theNamespace, string name, bool localeInvariant = false)
+        public virtual IEnumerable<TexlFunction> LookupFunctions(DPath theNamespace, string name, bool localeInvariant = false)
```

Comment:
>virtual [](http://example.com/codeflow?start=15&length=7)

@MikeStall this is a good change to make especially in light of discussions around composing resolvers

Reply:
Let's chat more - my concern is that we currently get the function list via PowerFxConfig. This means we have 2 ways to provide functions.